master-protected-branches/master.cfg

```diff
@@ -585,8 +563,8 @@ c["builders"].append(
 
 c["builders"].append(
     util.BuilderConfig(
-        name="amd64-fedora-40-last-N-failed",
-        workernames=workers["x64-bbw-docker-fedora-40-last-N-failed"],
+        name="amd64-rockylinux-9-last-N-failed",
```

Comment:
did we want to remove disto names out of builders doing very functional things like this and jepsen?